cmd/generate/config/spec.go

```diff
@@ -0,0 +1,142 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/charmbracelet/bubbles/help"
+	"github.com/charmbracelet/bubbles/key"
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+// Bubbletea for Interactive Mode
+
+type model struct {
+	textInput textinput.Model
+	help      help.Model
+	keymap    keymap
+
+	opts           *Options
+	attributionMap map[string][]string
+	uniqueEmails   []string
+	currentIndex   int
+}
+
+type keymap struct{}
+
+func (k keymap) ShortHelp() []key.Binding {
+	return []key.Binding{
+		key.NewBinding(key.WithKeys("ctrl+n"), key.WithHelp("ctrl+n", "next")),
+		key.NewBinding(key.WithKeys("ctrl+p"), key.WithHelp("ctrl+p", "prev")),
+		key.NewBinding(key.WithKeys("ctrl+i"), key.WithHelp("ctrl+i", "ignore email")),
+		key.NewBinding(key.WithKeys("esc"), key.WithHelp("esc", "quit")),
+		key.NewBinding(key.WithKeys("enter"), key.WithHelp("enter", "submit")),
+	}
+}
+
+func (k keymap) FullHelp() [][]key.Binding {
+	return [][]key.Binding{k.ShortHelp()}
+}
+
+func initialModel(opts *Options, uniqueEmails []string) model {
+	ti := textinput.New()
+	ti.Placeholder = "name"
+	ti.Focus()
+	ti.ShowSuggestions = true
+
+	return model{
+		textInput: ti,
+		help:      help.New(),
+		keymap:    keymap{},
+
+		opts:           opts,
+		attributionMap: make(map[string][]string),
+		uniqueEmails:   uniqueEmails,
+		currentIndex:   0,
+	}
+}
+
+func (m model) Init() tea.Cmd {
+	return textinput.Blink
+}
+
+func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	var cmd tea.Cmd
+	currentEmail := m.uniqueEmails[m.currentIndex]
```

Comment:
You're missing a check for the last attribution being ignored: If you attempt to ignore the last one, it panics on this line:

```
❯ pizza generate config . -i
Found email  - who to attribute to?:
> name

ctrl+n next • ctrl+p prev • ctrl+i ignore email • esc quit • enter submit
Caught panic:

runtime error: index out of range [1] with length 1

Restoring terminal...

goroutine 1 [running]:
runtime/debug.Stack()
        /usr/local/go/src/runtime/debug/stack.go:24 +0x64
runtime/debug.PrintStack()
        /usr/local/go/src/runtime/debug/stack.go:16 +0x1c
github.com/charmbracelet/bubbletea.(*Program).Run.func1()
        /Users/jpmcb/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.27.1/tea.go:511 +0x88
panic({0x100ec79a0?, 0x1400029ef18?})
        /usr/local/go/src/runtime/panic.go:770 +0x124
github.com/open-sauced/pizza-cli/cmd/generate/config.model.Update({{{0x0, 0x0}, {0x100cc14fc, 0x2}, {0x100cc1ad2, 0x4}, 0x0, 0x2a, {0x1f972880, {0x0, ...}, ...}, ...}, ...}, ...)
        /Users/jpmcb/workspace/opensauced/pizza-cli/cmd/generate/config/spec.go:68 +0x7d4
github.com/charmbracelet/bubbletea.(*Program).eventLoop(0x1400026f320, {0x100f125f8?, 0x1400030a008?}, 0x14000304180)
        /Users/jpmcb/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.27.1/tea.go:444 +0x630
github.com/charmbracelet/bubbletea.(*Program).Run(0x1400026f320)
        /Users/jpmcb/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.27.1/tea.go:582 +0x7ec
github.com/open-sauced/pizza-cli/cmd/generate/config.run(0x14000200ff0)
        /Users/jpmcb/workspace/opensauced/pizza-cli/cmd/generate/config/config.go:115 +0x280
github.com/open-sauced/pizza-cli/cmd/generate/config.NewConfigCommand.func2(0x1400029af08, {0x1400021b9a0?, 0x4?, 0x100cc19ce?})
        /Users/jpmcb/workspace/opensauced/pizza-cli/cmd/generate/config/config.go:67 +0x10c
github.com/spf13/cobra.(*Command).execute(0x1400029af08, {0x1400021b980, 0x2, 0x2})
        /Users/jpmcb/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:985 +0x840
github.com/spf13/cobra.(*Command).ExecuteC(0x1400029a308)
        /Users/jpmcb/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x344
github.com/spf13/cobra.(*Command).Execute(0x100e60dc0?)
        /Users/jpmcb/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041 +0x1c
main.main()
        /Users/jpmcb/workspace/opensauced/pizza-cli/main.go:17 +0x108
```